# -*- coding: utf-8 -*-
"""data_loader.ipynb

Automatically generated by Colab.

Original file is located at
    https://colab.research.google.com/drive/1JbtVn9uAiH_CYX3u17zI_v6NssPEWzFz
"""

# data_loader.py
import pandas as pd

def ler_dados(caminho: str) -> pd.DataFrame:
    try:
        return pd.read_csv(caminho)
    except Exception as e:
        raise RuntimeError(f"Erro ao ler dados: {e}")